### PR TITLE
Winter edition

### PR DESCRIPTION
**Pull request by edworczak**
Updated winter edition with improvements suggested by users.

## Description
Now the Winter Theme is disabled by default. To enable it go to `report/src/main/webapp/app/themes/winterEdition.directive.js` and change the value of `winterEditionEnabled` variable to `true`. Winter Theme will be visible 14 days before and after Christmas. Winter Theme contains:
- logo with Santa Claus's hat,
- new icons for tests,
- snow falling only on sidebar so it won't cover test results.

## Screenshot
winteredition

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [x] I have reviewed (and updated if needed) the documentation regarding this change

---
I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

File: report/src/main/webapp/app/themes/winterEdition.directive.js [hunk 1]
```diff
@@ -43,8 +43,10 @@ define(['angularAMD'], function (angularAMD) {
       var isChristmas = isDateWithinMargin(TODAY, CHRISTMAS_DATE, DAY_MARGIN);
 
       if (isChristmas) {
```
Comment (Skejven) [inline report/src/main/webapp/app/themes/winterEdition.directive.js]: Could we please add additional flag, e.g. `winterEditionEnabled` and make the condition:
```
if (winterEditionEnabled && isChristmas) {
```
?
It should be set to `false` by default.
If admin of an AET instance decides to enable it, it should be manually set to `true` on the server instance.

Comment (edworczak) [reply report/src/main/webapp/app/themes/winterEdition.directive.js]: Done.

File: report/src/main/webapp/app/themes/winterEdition.directive.js [hunk 2]
```diff
@@ -42,9 +45,11 @@ define(['angularAMD'], function (angularAMD) {
       var DAY_MARGIN = 14;
       var isChristmas = isDateWithinMargin(TODAY, CHRISTMAS_DATE, DAY_MARGIN);
 
-      if (isChristmas) {
+      if (isChristmas && winterEditionEnabled) {
```
Comment (Skejven) [inline report/src/main/webapp/app/themes/winterEdition.directive.js]: Please revert the order of flags.

Comment (edworczak) [reply report/src/main/webapp/app/themes/winterEdition.directive.js]: Done.

**Comment by edworczak** [comment]
Added description and screenshot to PR.

**Pull request merged**